app/views/shared/check_answers/_merits_proceeding_section.html.erb

```diff
@@ -1,4 +1,6 @@
 <% @legal_aid_application.proceedings.each do |proceeding| %>
+  <% next if @legal_aid_application.legal_framework_merits_task_list.task_list.tasks.dig(:proceedings, proceeding.ccms_code.to_sym)[:tasks].empty? %>
```

Comment:
perhaps a helper

```
#
def mertis_task_list_empty_for?(proceeding_ccms_code)
  legal_framework_merits_task_list.task_list.tasks.dig(:proceedings, proceeding_ccms_code.to_sym)[:tasks].empty?
end
```

Reply:
As these are due to be replaced as the CYA refresh - we could do it then?

Reply:
Had second thoughts... moved it into the task_list_helper, can I get a re-review?